ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -816,6 +816,8 @@ redundancy:
 
 ```yaml
 hardware:
+  access_list:
+    mechanism: < algomatch | none | tcam | default -> algomatch >
```

Comment:
the default and supported values in EOS is highly platform dependent, so I would prefer not to have any default in `eos_cli_config_gen`. Just configure whatever value the user specifies, or nothing. This is already what you do in the template, so just remove the default from the doc.